lib/v2/ccreports/index.js

```diff
@@ -0,0 +1,46 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootUrl = 'https://www.ccreports.com.cn';
+
+module.exports = async (ctx) => {
+    const listData = await got.get(rootUrl);
+    const $ = cheerio.load(listData.data);
+    const list = $('div.index-four-content > div.article-box')
+        .find('div.new-child')
+        .map((_, item) => ({
+            title: $(item).find('p.new-title').text(),
+            link: new URL($(item).find('a').attr('href'), rootUrl).href,
+            author: $(item)
+                .find('p.new-desc')
+                .text()
+                .match(/作者：(.*?)\s/)[1],
+            pubDate: parseDate(
```

Comment:
Please parse the pubDate from the article. It has a full date and time.

Reply:
OK